src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import taskitems.Greet;
+import taskitems.exceptions.IllegalInputException;
+import taskitems.TaskManager;
+
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
 public class Duke {
+
+    static Greet greet;
+    static Scanner reader = new Scanner(System.in);
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        TaskManager taskManager = new TaskManager();
+        try {
+            taskManager.loadData();
+        } catch (FileNotFoundException f) {
+            System.out.println("No saved data found");
+        }
+        greet.sayHi();
+        taskLoop(taskManager);
+        //echo();
```

Comment:
You could remove unused code like this line to improve overall code readability.